src/main/java/org/jboss/ejb/client/EJBClientContext.java

```diff
@@ -475,6 +487,20 @@ public void run() {
         }
     }
 
+    public void unregisterNodeEJBReceivers(String nodeName) {
```

Comment:
Or better, similar to this : https://github.com/jbossas/jboss-ejb-client/blob/2.1/src/main/java/org/jboss/ejb/client/ClusterContext.java#L414